src/Microsoft.DotNet.Build.Tasks/PackageFiles/crossgen.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+__TargetFramework=${3:-}
```

Comment:
Shouldn't parameter checking go with the rest of the parameter checking below on line 84?